examples/seismic/acoustic/accuracy.ipynb

```diff
@@ -240,9 +250,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 12,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Numerical data min,max,abs; -5.349670e-03 +8.529989e-03 +8.529989e-03\n",
+      "Analytic  data min,max,abs; -5.322232e-03 +8.543911e-03 +8.543911e-03\n"
+     ]
+    }
+   ],
    "source": [
     "print(\"Numerical data min,max,abs; %+12.6e %+12.6e %+12.6e\" % \n",
     "      (np.min(ref_rec.data), np.max(ref_rec.data), np.max(np.abs(ref_rec.data)) ))\n",
```

Comment:
I think this needs to be `1.2%e` to avoid issue with different compilers in output value.

Reply:
see below, added #ignore